### ISSUE TITLE
Install error, affects pngquant-bin as well

### ISSUE THREAD

**Issue by scjackson** [imagemin/jpegtran-bin#77]
Getting the same error for both packages, both were installing fine yesterday however not anymore today (I didn't change any deps).

```
> jpegtran-bin@3.1.0 postinstall /.../node_modules/jpegtran-bin
> node lib/install.js

module.js:341
    throw err;
    ^

Error: Cannot find module './src/init'
```

**Comment by karlito-who-else** [comment]
Having run the `npm install` and `yarn` commands on my project a few times (deleting the node_modules folder between commands and clearing the cache) I have found that  I have been able to successfully install the dependencies in the project's package.json via npm 3.10.8 on every attempt but encountered this error occasionally when installing via yarn 0.15.1.  That said, given the inconsistency of a successful installation, I feel as if I would encounter this issue if I made further attempts at an `npm install`, though that is just speculation on my part.

I'm using macOS 10.12 and have installed node 6.8.0 via Homebrew.

I'm not including jpegtran-bin directly in my package.json but I am instead including imagemin-jpegtran 5.0.2 so apologies if my experience is not relevant to the issue.
